<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainInterface</class>
 <widget class="QWidget" name="MainInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="Off_Online_Game">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/resource/danji.png);
background-color: rgb(255, 255, 255,0.5);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Online_Game">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/resource/pipei.png);
background-color: rgb(255, 255, 255,0.5);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>570</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/resource/tuichu.png);
background-color: rgb(255, 255, 255,0.5);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="onlinePlayer">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>80</y>
     <width>330</width>
     <height>240</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,0.5);</string>
   </property>
  </widget>
  <widget class="QListWidget" name="MsgBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>390</y>
     <width>330</width>
     <height>220</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,0.5);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sendText">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>620</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,0.5);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendMsg">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>620</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="invitation">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>330</y>
     <width>120</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/resource/yaoqing.png);
background-color: rgb(255, 255, 255,0.5);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="watchGame">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>330</y>
     <width>120</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/resource/see.png);
background-color: rgb(255, 255, 255,0.5);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="gameRecord">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>490</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/resource/fupan.png);
background-color: rgb(255, 255, 255,0.5);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>670</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/resource/back.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>20</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resource/tuichu2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="min">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>20</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resource/suoxiao.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="GameRecordBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>331</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseGameRecord">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resource/tuichu2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="qingding">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>520</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resource/sure.png);
background-color: rgb(255, 255, 255,0.5);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="user_infor">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,0.5);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/resource/zaixian.png</pixmap>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>Off_Online_Game</zorder>
  <zorder>Online_Game</zorder>
  <zorder>exit</zorder>
  <zorder>onlinePlayer</zorder>
  <zorder>MsgBox</zorder>
  <zorder>sendText</zorder>
  <zorder>sendMsg</zorder>
  <zorder>invitation</zorder>
  <zorder>watchGame</zorder>
  <zorder>gameRecord</zorder>
  <zorder>close</zorder>
  <zorder>min</zorder>
  <zorder>GameRecordBox</zorder>
  <zorder>CloseGameRecord</zorder>
  <zorder>qingding</zorder>
  <zorder>user_infor</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
